2021-09-01 09:48:26.751 -04 [2980] ERROR:  llave duplicada viola restricción de unicidad «ir_model_fields_name_unique»
2021-09-01 09:48:26.751 -04 [2980] DETALLE:  Ya existe la llave (model, name)=(project.task, x_viatico).
2021-09-01 09:48:26.751 -04 [2980] SENTENCIA:  INSERT INTO "ir_model_fields" ("id", "create_uid", "create_date", "write_uid", "write_date", "compute", "copied", "depends", "domain", "field_description", "help", "index", "model", "model_id", "name", "on_delete", "readonly", "related", "required", "selectable", "state", "store", "tracking", "ttype") VALUES (nextval('ir_model_fields_id_seq'), 2, (now() at time zone 'UTC'), 2, (now() at time zone 'UTC'), NULL, true, NULL, '[]', 'Viatico', 'Viatico necesario', false, 'project.task', 537, 'x_viatico', 'set null', false, NULL, false, true, 'manual', true, 0, 'monetary') RETURNING id
